academy/2-cosmos-concepts/7-multistore-keepers.md

```diff
@@ -238,8 +263,40 @@ func (k Keeper) GetStoredGame(ctx sdk.Context, gameId string) (storedGame types.
 }
 ```
 
+</CodeGroupItem>
+<CodeGroupItem title="v0.50 or after">
+
+```go
+package keeper
+
+type Keeper struct {
+    ...
+    StoredGameList collections.Map[string, checkers.StoredGame]
```

Comment:
should mention this is an alternative way not the only way

Reply:
I left a suggestion below for that.